check-diff.sh

```diff
@@ -565,15 +564,16 @@ function run_phpunit_local {
 	(
 		echo "## phpunit"
 		if [ -n "$( type -t phpunit )" ] && [ -n "$WP_TESTS_DIR" ]; then
-			if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
+			if [ -n "$PHPUNIT_CONFIG" ]; then
```

Comment:
@MattGeri I think that should work actually. The two scenarios I am think of for running PHPUnit tests are:

1. You have repo for a theme or plugin project which has a `phpunit.xml` or `phpunit.xml.dist` in its root. There wouldn't be nested unit test suites in this case.
2. You have a repo containing an entire site project, with `wp-content` and all, and in this case there would be no root `phpunit.xml` config, but there _would_ be nested unit test suites.

@valendesigns am I missing another scenario?